TombEngine/Game/effects/tomb4fx.cpp

```diff
@@ -866,6 +866,10 @@ void TriggerGunShell(short hand, short objNum, LaraWeaponType weaponType)
 			offset = Vector3i(8, 48, 40);
 			break;
 
+		/*case LaraWeaponType::Magnums:
```

Comment:
pls don't comment code like that if possible, if it needs more stuff fixed etc then make a TODO comment above it. Otherwise just remove

Reply:
I will remove that part of code, I used for initial experiments to understand the engine. Thanks for letting me know.